src/haproxy/services/utils.sh

```diff
@@ -0,0 +1,84 @@
+source /var/vcap/packages/haproxy/utils.sh
```

Comment:
this script seems to do things with Warden. How is this related to haproxy? Is this the script that was added from common package? Is the only functionality it provides sourcing real haproxy script?